src/main/java/edu/cornell/kfs/concur/businessobjects/ConcurAccountInfo.java

```diff
@@ -64,4 +76,27 @@ public void setProjectCode(String projectCode) {
     public void setOrgRefId(String orgRefId) {
         this.orgRefId = orgRefId;
     }
+    
+    @Override
+    public String toString() {
+        StringBuffer result = new StringBuffer();
```

Comment:
You may want to use StringBuilder here instead of StringBuffer.  StringBuilders are thread safe, so if this is single threaded, StringBuilder will perform a bit better.  Chad H suggested I start using StringBuilder.